src/main/external-resources/renderers/Seiki-SmartTV.conf

```diff
@@ -0,0 +1,766 @@
+#----------------------------------------------------------------------------
```

Comment:
I removed it in the next commit because I don't know what else to do. I'm still figuring out how to work with git. I will push it later on.

Reply:
Ok great, thanks. If you have questions about Git let us know and we can probably help. In this case you are submitting your changes to us via your `master` branch, so I recommend using a different branch for each feature, e.g. a `SeikiSmartTV` branch that just has that config file in it. It can be hard to learn Git at first, we have all been through it